src/client/Microsoft.Identity.Client/Instance/AadAuthority.cs

```diff
@@ -51,10 +57,10 @@ internal static bool IsCommonOrganizationsOrConsumersTenant(string tenantId)
                 s_tenantlessTenantNames.Contains(tenantId);
         }
 
-        internal override string GetTenantedAuthority(string tenantId)
+        internal override string GetTenantedAuthority(string tenantId, bool forceTenantless = false)
```

Comment:
What does forceTenantless represent here? How is it different from /common or /organizations which are also tenantless?

Reply:
"l.m.o/12345-1234-1234".GetTenantAuthority("common", false)  returns "l.m.o/12345-1234-1234"\
"l.m.o/12345-1234-1234".GetTenantAuthority("common", true)  returns "l.m.o/common"

Reply:
I would rename it to something like `useSpecifiedTenant` or `forceSpecifiedTenant`. Since I can call this method as `GetTenantedAuthority("12345-1234-1234", forceTenantless: true)` and it will return me a tenanted authority.

Also I would add some authority unit tests that test that new parameter.